### DISCUSSION TITLE
4기 회고 20210329

### DISCUSSION THREAD

**Discussion by imakerjun** [next-step/blog#201] [회고]
# 📅 3월 한달간의 회고

**Comment by oen142** [comment]
### 3월 한달간 인사이트 있었던 일
1. AWS Certification 모임 리드 시작한 일
2. AWS에 대해서 좀더 깊이 공부할 수 있는 한달
3. 네트워크와 인프라(devops)에 대해서 좀더 공부할수 있었다.
4. 세션관리 어떻게 해야하는지 좀더 알수 있었음(큰 수확)
### 3월 한달간 아쉬움이 남은 일
1. 자바 백엔드 개발 공부 부족하게 한것
2. 이직준비를 위해 이력서 작성하기로 했으면서 나중으로 미룸
3. 역시 개인 프로젝트를 리딩하는것이 쉽지가 않았다.(중간에 파기됨)
4. 주간 회고 작성하자고 했는데 귀찮아서 미룸(글만 쓰려고 하면 왜 이렇게 귀찮아 지는걸까)
### 4월에 다르게 하고 싶은 것이 있다면?
1. 이력서 정리하기
2. DVA-01 꼭 따자 
3. 블로그 글 적기 귀찮아 하는것을 고치자

**Reply by imakerjun** [reply]
**프로 정리러 👍🏻**
개인 프로젝트 리딩하는 것이 중간에 파기되었다고 해주셨는데 어떤 점때문에 파기되었을까요? 😭

**Comment by imakerjun** [comment]
# 📅 3월 한달간의 회고
## 💡 3월 한달간 인사이트 있었던 일
- 나의 역량별로 그래프와 과거 경험들을 정리해본일
  - 그리고 그 경험을 토대로, 학생들과 면담하면서 함께 성장할 수 있는 계기와 템플릿을 제공한 일
- 학생들과 면담하면서 과정에 대한 만족 피드백을 받은 일 
- 블로그 스터디 PR을 플랫폼으로 만들어서 시작한 일
- 레벨1스터디에서 온라인 세션 시간에 실시간 코드리뷰를 시작한 일

## 🤔 3월 한달간 아쉬움이 남은 일
- 블로그 스터디 플랫폼에 사용자가 참여할 수 있는 기능이 없다는 점. 구독 기능을 추가하고 싶었는데 아직 넣지 못했다 😭 
- 크루들과의 대화와 커뮤니케이션 과정을 잘 기록하지 못한 일

## 🪜 4월에 다르게 하고 싶은 것이 있다면?
- 4월에는 블로그 스터디 플랫폼 구독기능을 추가하여, 구독자 50명이상 만들기!
- 크루들과의 레벨1과정에 대한 후기 및 회고 글 작성하기
  - 대화를 하며 인사이트 얻은 일
  - 내게서 부족함을 느낀 일
  - 그랬기에 더 노력하려고 했고, 다가가기 위해 했던 시도들

**Reply by oen142** [reply]
구독 기능 추가하면 금방 늘어날것 같은데 얼른 추가해주세요!

**Comment by catsbi** [comment]
### 💡 3월 한달간 인사이트 있었던 일
1. ATDD과정을 진행하면서 작년에 참가했던 TDD과정에서 겪은 테스트방식이나 이런 부분들을 실제 실무에서 어떻게 적용하면 좋을지에 대한 인사이트를 얻고 좀 더 다양한 디자인패턴도적용해볼 수 있는 기회였다. 
2. 스프링을 기초부터 다시 공부하면서, 내가 공부는 꾸준히 했지만 기본기를 너무 자바나 자바스크립트같은 언어기반 공부에만 치중했던 것 같다는 생각을 했고, 스프링 프레임워크 자체에 대해 기본기를 다지면서 내가 쓰는 어노테이션들이나 스프링에서 프레임워크가 어떻게돌아가고 왜 이렇게 돌아가는지에 대해 학습할 수 있었다. 

### 🤔 3월 한달간 아쉬움이 남은 일
1. 년초부터 필요에 의해 학습했던 부분들(Swify, Flutter, Dart) 작년에 학습하던 부분들(CleanCode, Hyper Focus...)을 제대로 마무리 짓지 못했다는점. 
2. 중간중간 핑계대면서 빈둥거린 시간도 꽤 많았다는 부분

### 🪜 4월에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
1. 구매하고 못 본 강의들 다 마무리 짓기
2. 이력서 작성및 멘토링 신청해보기
3. (DDD과정이 열린다면) 해당 과정 수강하고 이번에도 수료하기

**Reply by oen142** [reply]
`구매하고 못 본 강의들 다 마무리 짓기` 역시 국룰입니다.

**Reply by inhyuck** [reply]
>중간중간 핑계대면서 빈둥거린 시간도 꽤 많았다는 부분 

역시 공감입니다.

**Reply by imakerjun** [reply]
4월에 못 본 강의 진행상황 여쭤보기!! ㅎㅎ

**Comment by bigsaigon333** [comment]
## 💡 3월 한달간 인사이트 있었던 일
- 우테코에서의 하루하루가 인사이트의 연속이었다. 
- 클럽하우스에서 라인 개발자들에게 아무거나 물어보세요 라는 코너를 들은 적이 있다. 20년, 30년 뒤에도 개발을 하고 싶은데 그런 과정 중에 열정이 식거나, 번아웃에 빠지는 일도 있을거 같은데 어떻게 이겨낼 수 있을지 질문을 드렸었다. 답변으로는, 키보드 앞에 있는 것만이 프로그래밍이 아니다. 라는 말씀을 해주셨다. 

## 🤔 3월 한달간 아쉬움이 남은 일
- 시간관리가 되지 않았다. 아무런 계획 없이 닥치는대로 개발하고, 블로그 스터디 할 때쯤 부랴부랴 글로 정리하고, 정말 필이 꽂히는 대로 지냈던 것 같다. 생활패턴이 불규칙해져서 항상 피곤하고, 건강이 안좋아진 것 같다.

## 🪜 4월에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)

꾸준하게 개발하기
- 예를 들어 일주일에 7시간 사이드 프로젝트를 진행한다고 가정하였을 때, 3월에는 일주일 중에 하루에 몰아서 7시간을 개발한 것 같다. 4월에는 매일매일 1시간씩 개발하는 방향으로 하고 싶다.

규칙적인 생활하기
- 취침시간, 기상시간을 일정하게 유지한다.
- 일정 시간 이후로는 개발을 하지 않는다. 개발에 대한 열정을 남겨둔 채로 잠자리에 든다
- 운동을 한다(피티샵 컴백하기!)

**Reply by oen142** [reply]
`키보드 앞에 있는것만이 프로그래밍이 아니다` 명언이네요! 
저도 최근에 계속 코딩욕심만 생겨서 앉아있는 시간만 늘렸는데 좀더 움직여야겠군요

**Comment by inhyuck** [comment]
# 📅 3월 한달간의 회고
## 💡 3월 한달간 인사이트 있었던 일
- 블랙커피 커뮤니티(?)에서 블로그 스터디 & 프론트엔드 스터디 진행
- 가족 행사

## 🤔 3월 한달간 아쉬움이 남은 일
- 소중한 사람과 함께하는 시간에 집중하지 못한 일 
- 집중해야 하는 시간에 집중하지 못하고 후회하며 무의미한 시간을 보낸 일

## 🪜 4월에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
- 꽃 구경 🌸
- 책 읽기 (기술서적 + @)
- 프론트엔드 스터디 Lv2 진행 || 클린 리액트 프로그래밍

**Reply by oen142** [reply]
꽃구경 이번에 벚꽃시즌이라 가면 좋을것같아요!

**Comment by khw970421** [comment]
1) 3월 한달간 인사이트 있었던 일
 ● 조코딩 강의를 통해 나만의 오버워치 mbti를 만들고 이를 홍보함으로써 다같이 즐겨준 것

2) 3월 한달간 아쉬움이 남은 일
 ●  4학년이되어서 막상 내년이면 내실력으로 취업할수있을까 하는 불안감과 스트레스의 압박으로 무엇을 우선시해야할지 모르겠는 막연함
 ●   따로 js-study를 하는데 코드스피츠에서 하는 객체지향강의에 mvc, mvp, mvvm 코드들이 너무 어려워 봐도봐도 제대로 이해 못하고 있는 상황

3)  4월에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
● 오버워치 mbti 만든것 좀 더 다듬기
● 프레임워크 없는 프론트엔드 책 읽어보기

**Reply by oen142** [reply]
저도 불안감과 스트레스압박이 너무 심했는데 언제나 다들 같은 마음입니다 ㅠㅠ

**Comment by yujo11** [comment]
# 📅 3월 한달간의 회고

### 💡 3월 한달간 인사이트 있었던 일

1. 그동안 신경쓰지 못 했던 웹 접근성, HTML/CSS의 중요성을 깨달았다.
   - '웹'에 대해 다시 생각해보게 됐다.
   - JS뿐만 아니라 HTML/CSS도 잘하고 싶다는 욕심이 생겼다.
2. '잘' 하는 것에 신경을 많이 썼는데 '잘' 아는 것도 중요하다는걸 깨달았다.
   - 단순히 '잘'하기만 하는 개발자가 아닌 '잘' 하고 '잘' 아는 개발자가 되고싶다.

### 🤔 3월 한달간 아쉬움이 남은 일

1. 독서를 하지 못한 점.
   - 2월 말에 도서를 몇 권 구입했는데 아직 한권도 읽지 못 했다.
2. 마음 속 조급함을 버리지 못 한 점.
   - 내 페이스를 유지하지 못 하고 조급함에 쫓겨다니기만 했다.

### 🪜 4월에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)

1. '남'보다는 '나'에 집중하며 페이스 조절하기
   1. 남과 비교하지 않는다.
   2. 하루 중 1시간이라도 내가 하고 싶은 공부를 하는 시간을 갖는다.
   3. 기상 후 30분 이내의 가벼운 운동을 한다.
2. 꾸준히 독서하기
   1. 책을 항상 휴대한다.
   2. 시간이 있을 때마다 핸드폰 대신 책 읽기
   3. 2주에 한 권을 목표로 하기.
   4. 독서 후 블로그에 서평 남기기.

**Reply by oen142** [reply]
`2월 말에 도서를 몇 권 구입했는데 아직 한권도 읽지 못 했다`
저랑 같이 다음달은 도서 한권읽기 도전해봐요!

**Reply by imakerjun** [reply]
저랑 동동님이랑 같이 올림픽 공원 산책가요 ㅎㅎ

**Reply by imakerjun** [reply]
4월에 함께 자라기 서평 작성할 예정이시군요!

**Reply by khw970421** [reply]
`마음 속 조급함을 버리지 못 한 점` 공감되네요
